crates/egui_demo_lib/src/demo/plot_demo.rs

```diff
@@ -823,6 +841,7 @@ impl InteractionDemo {
             pointer_coordinate_drag_delta.x, pointer_coordinate_drag_delta.y
         );
         ui.label(format!("pointer coordinate drag delta: {coordinate_text}"));
+        ui.label(format!("hovered plot item: {hovered_plot_item:?}"));
```

Comment:
Let's go all the way and print if it was "sin" or "cos" being hovered (or neither)